lib/openstudio/workflow/jobs/run_reporting_measures.rb

```diff
@@ -62,9 +62,23 @@ def perform
 
     # Ensure output_attributes is initialized in the registry
     @registry.register(:output_attributes) { {} } unless @registry[:output_attributes]
+    
+    #get OSA[:urbanopt]  #BLB should prob be sent in as cli arg and used in options but for now just do this
+    @registry.register(:urbanopt) { false }
+    if @registry[:osw_path]
+      workflow = nil
+      if File.exist? @registry[:osw_path]
+        workflow = ::JSON.parse(File.read(@registry[:osw_path]), symbolize_names: true)
+        if !workflow.nil?
+          if !workflow[:urbanopt].nil?
+            @registry.register(:urbanopt) { workflow[:urbanopt] }
+          end
+        end
+      end    
+    end
 
     # Load simulation files as required
-    unless @registry[:runner].halted
+    unless @registry[:runner].halted || @registry[:urbanopt]
```

Comment:
so when running urbanopt these is no catch for when it stops?

Reply:
this is for the --postprocess run through the OS CLI, so that urbanopt can run its reporting measures.  The runner might still be active, but there is no idf, osm, epw or sql file and so the raise commands below will fail.  This just bypasses those checks if you have urbanopt set to true., which is defaulted to false.

Reply:
I see... that makes sense, thanks.